patches/tModLoader/Terraria/ModLoader/Config/UI/UIModConfig.cs

```diff
@@ -395,7 +395,7 @@ public override void Draw(SpriteBatch spriteBatch)
 		base.Draw(spriteBatch);
 
 		if (!string.IsNullOrEmpty(Tooltip)) {
-			UICommon.DrawHoverStringInBounds(spriteBatch, Tooltip, GetDimensions().ToRectangle());
+			UICommon.MouseText(Tooltip, true);
```

Comment:
Always use named parameters (`hoverBackground: true`) when passing raw bool or integer values to methods where the meaning of the param is not obvious